src/main/java/it/smartcommunitylab/aac/config/AACConfig.java

```diff
@@ -2,8 +2,16 @@
 
 import java.io.IOException;
 import java.io.Writer;
+
 import javax.sql.DataSource;
 
+import com.fasterxml.jackson.annotation.JsonInclude.Include;
```

Comment:
fix changes to avoid moving around existing code

Reply:
Fixed in 1a508e9db3be04a6e96019f3cc87784f6824c5dd